packages/common/src/discussions/edit.ts

```diff
@@ -126,13 +127,34 @@ export async function updateDiscussion(
   let updatedDiscussion = mapper.storeToEntity(updatedModel, discussion);
   updatedDiscussion = computeProps(model, updatedDiscussion, requestOptions);
 
+  // persist location geometries to discussion settings as Geometry[]
+  let allowedLocations;
+  try {
+    allowedLocations =
+      updatedDiscussion.location?.geometries?.map((geometry) =>
+        arcgisToGeoJSON(geometry)
+      ) || null;
+  } catch (e) {
+    /* istanbul ignore else */
+    if (e instanceof Error) {
```

Comment:
I'm curious why this is needed? Can `arcgisToGeoJson` throw a non-error value (e.g. a `string`, suspect related to [this line](https://github.com/Esri/hub.js/pull/1316/files#diff-e52ff993e5b69ce6496000695934c6bbafe6dd867d76e42052486a335836188fR176))? If so, is the full array of geometries successfully converted to geojson when that occurs? `allowedLocations` will be `undefined`, curious if it should be `null`?

Additionally, the way this logic is written, `allowedLocations` could be one of `undefined`, `null` or `Geojson[]`, is that intentional? I would think it would be either `null` or `Geojson[]`?